### ISSUE TITLE
NVDA reads selected item twice in Chrome when DropDownList filter is enabled

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4763]
### Description

When using a DropDownList with the filter feature enabled, NVDA screen reader announces the selected list item twice in Chrome. The issue does not occur in other browsers. In Firefox, it reads the selected item once.

### Steps To Reproduce

Open the following [example](https://demos.telerik.com/kendo-angular-ui/demos/dropdowns/dropdownlist/filtering/built_in_directive?theme=default-ocean-blue-a11y) from our documentation and use NVDA.

### Screenshots or video

https://github.com/user-attachments/assets/b0ae46b5-e585-4d84-aa0f-f508e5a19a93

<img width="255" height="245" alt="Image" src="https://github.com/user-attachments/assets/ac255fc7-8431-494c-a426-39fdf93ebe9b" />

### Expected Behavior

To read the selected item once, like in Firefox.
<img width="258" height="169" alt="Image" src="https://github.com/user-attachments/assets/0874a12f-2638-442e-9033-82a2b5f850fa" />

**Comment by georgi-sla** [comment]
The Kendo UI for Angular components are designed to comply with official accessibility standards, but cannot address missing features or non-standard behaviors in existing accessibility technologies. For reference, here is our list of supported screen readers and browser combinations:

- https://www.telerik.com/kendo-angular-ui/components/accessibility#screen-readers

I have tested the DropDownList component using the supported combinations:
- JAWS screen reader with Chrome
- NVDA screen reader with Firefox

Based on this testing, the component behaves as expected and meets accessibility requirements.
